components/style/properties/longhand/font.mako.rs

```diff
@@ -768,6 +768,10 @@ ${helpers.single_keyword_system("font-variant-caps",
                         return Some(em)
                     }
                 }
+            } else if let SpecifiedValue::Larger = *self {
+                return Some(1.2)
+            } else if let SpecifiedValue::Smaller = *self {
+                return Some(1./1.2)
```

Comment:
I left two comments on the bug, but I'm actually surprised tidy doesn't complain about the lack of spacing here around the `/`.